src/ctime.js

```diff
@@ -151,6 +178,7 @@ const ctime = (t, f) => {
     api.set = afunc(set, api);
     api.add = afunc(add, api);
     api.subtract = afunc(subtract, api);
+    api.startOf = afunc(startOf, api);
```

Comment:
/*assignTime*/ should be true otherwise the mutation chain will break

```suggestion
    api.startOf = afunc(startOf, api, /*assignTime*/true);
```